x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -658,6 +658,20 @@
     "core.euiSuperUpdateButton.refreshButtonLabel": "更新",
     "core.euiSuperUpdateButton.updateButtonLabel": "更新",
     "core.euiSuperUpdateButton.updatingButtonLabel": "更新中",
+    "kibana-react.table_list_view.listing.deleteButtonMessage": "{itemCount} 件の {entityName} を削除",
```

Comment:
I think the ids should use camel case: `kibana-react.tableListView.listing.deleteButtonMessage `